server/src/main/java/com/cloud/alert/AlertManagerImpl.java

```diff
@@ -718,6 +718,11 @@ public void sendAlert(AlertType alertType, long dataCenterId, Long podId, Long c
         mailProps.setContent(content);
         mailProps.setContentType("text/plain");
 
+        if (recipients == null) {
+            s_logger.info("No recipients set in 'alert.email.addresses', skipping sending an alert");
```

Comment:
We could add subject and content to the message, e.g. `... skipping sending alert with subject [%s] and content [%s].`  
  Also, IMHO, I think it would be better to log it as `warning`.